docs/run-instance/data.md

```diff
@@ -29,6 +29,21 @@ Data relating to the avoid borders features is derived from administrative bound
 Configuration parameters: [
 `ors.engine.profiles.<PROFILE-NAME>.ext_storages.Borders`](configuration/engine/profiles/build.md#borders)
 
+To ensure `avoid_borders` functions correctly, the following are required:
+
+- **Field names and encoding:**  
+  `borders.geojson` must include a `name` property.  
+  In `ids.csv`, the `name` column must use consistent Latin/ASCII names.  
+  Names must match across `borders.geojson`, `ids.csv`, and `openborders.csv`.
```

Comment:
Apparently countries in **openborders.scv** are identified by their English names, as suggested by the following entries.

**ids_iso.csv**
```
...
"117","Lëtzebuerg","Luxembourg","LU","LUX"
...
```

**openborders.csv**
```
...
"Luxembourg","Austria"
"Luxembourg","Belgium"
"Luxembourg","Czech Republic
...
```

Reply:
Rewritten to accommodate for these details.